app/controllers/questions_controller.rb

```diff
@@ -10,28 +10,53 @@ def index
     @question_limitations = ENV["limit_question_index_to_users_questions_only"]
   end
 
+  def show
+    @questions = get_question_list()
+    @question_id = params[:id]
+    @question = @questions[(params[:id]).to_i - 1]["text"]
```

Comment:
This should also change, but it will require view changes too
```
@question = @questions[(params[:question_id]).to_i - 1]
@question_text = @question["text"]
```
Then the logger statements can use `@question`.

The view would have to change to ues `@question_text` though instead of `@question`.